Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
-
-
+function addTWo(num1,num2){
+    return num1+num2;
+}
+console.log(addTWo(3,7))
```

Comment:
The variable 'greeting' should be camelCased as 'greeting' to match the standard naming conventions.